src/js/components/stock-movement-wizard/fullOutbound/FullOutboundImport.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+
+import FullOutboundImportConfirm from 'components/stock-movement-wizard/fullOutbound/FullOutboundImportConfirm';
+import FullOutboundImportDetails from 'components/stock-movement-wizard/fullOutbound/FullOutboundImportDetails';
+import WizardStepsV2 from 'components/wizard/v2/WizardStepsV2';
+import useWizard from 'hooks/useWizard';
+
+const FullOutboundImport = () => {
+  const FullOutboundImportStep = {
+    DETAILS: {
+      key: 'DETAILS',
+      // TODO: Make titles translatable in OBPIH-6329
+      title: 'Create',
+    },
+    CONFIRM: {
+      key: 'CONFIRM',
+      title: 'Confirm',
+    },
+  };
+  const steps = useMemo(() => [
+    {
+      key: FullOutboundImportStep.DETAILS.key,
+      title: FullOutboundImportStep.DETAILS.title,
+      Component: () => (<FullOutboundImportDetails />),
+    },
+    {
+      key: FullOutboundImportStep.CONFIRM.key,
+      title: FullOutboundImportStep.CONFIRM.title,
+      Component: () => (<FullOutboundImportConfirm />),
+    },
+  ], []);
+
+  const stepsTitles = steps.map((step) => ({
+    title: step.title,
+    key: step.key,
+  }));
+
+  const [
+    Step,
```

Comment:
Can you get `Component` and `key` by destructuring?
I would also like to make the object returned from the hook flat. It doesn't make sense to add an additional object.
``` 
return [
    Step,
      set,
      first,
      next,
      previous,
      last,
  ];
```

Reply:
imho it's cleaner to distinguish the step and the util hook functions, this is why I made it like this.